### ISSUE TITLE
Testing for Research Outputs tab and ID minting feature

### ISSUE THREAD

**Issue by mariapraetzellis** [CDLUC3/dmptool#249]
**Project Details** 
- [x] On the project details page if you enter the project dates incorrectly and then try to save you are directed to a error message but loose all your work. Can we somehow get the error earlier or save the other data fields? - [x] Maria
- [x] The Project abstract box is missing the TinyMCE editor - [x] Maria
- [x] The Funding Opportunity Number field is defaulting to a value of "12345" (not seeing this anymore) - [x] Maria
- [x] The Funder ROR selector on the Project Details page does not appear for me. I see a flash for a moment, but then it disappears. - [x] Maria

**Collaborators** 
- [x] The "Add a Contributor" button is missing on the Collaborators tab - [x] Maria
![screenshot-dmptool-stg.cdlib.org-2021.03.11-12_00_20.png](https://images.zenhubusercontent.com/5f04b83738cdcf7d7c00e2e1/c126b406-493b-40ea-8de9-2281a1a50539)

**Research Outputs**
- [x] "The Research Outputs" tab is missing the add output button  - [x] Maria
![screenshot-dmptool-stg cdlib org-2021 03 11-12_16_44](https://user-images.githubusercontent.com/27511575/110849209-d0b79f00-8263-11eb-8f86-6eb355e8a54a.png)

- [x] The license field has way too many options coming from the SPDX list. I chatted w/ Erin and this is the best list for licenses so we're in the right place, but still need to modify since it's way too long right now. Can we pull out the generic [Creative Commons options](https://creativecommons.org/about/cclicenses/)? We can then give folks an "Other" option that then pulls up the full SPDX list. Let's talk about this and see if it makes sense in practice. - [x] Maria
- [x] Change the display text for the selected license to the title of the license- linked to the record in SPDX. [x] Maria
- [x] Remove the tool tip for Sensitive Data. I haven't been able to find a succinct definition that is actually useful so I think we should skip this for now. We can add a tool tip in later if needed. [x] Maria
- [x] If you select one of the additional (non CC-BY) licenses it correctly saves and updates the Hub but does not show it when you revisit the edit Research Output page
- [x] Remove the tool tip for Initial Access Level. It is too long as is, and I think that it's pretty self-explanatory and hopefully doesn't need a tip. [x] Maria
- [x] Enable the feature to include Research Outputs data with the downloaded DMP. [x] Maria

**Issues related to ID Minting** 
- [x] Checking the metadata within the DMPTool against the metadata in the XML record sent to DataCite I noticed a problem with contributors. The plan here: https://dmptool-stg.cdlib.org/plans/58899/contributors has three contributors listed, two of which are UCNRS and one is UCB. The [XML file](https://ezid.cdlib.org/manage/display_xml/doi:10.5072/FK2V69J19C) lists the same contributor twice and is grabbing the wrong ORCID. 
- [x] This might just be an issue with a bad ROR, but I am the creator of [this plan ](https://dmptool-stg.cdlib.org/plans/58899) but in the [XML record](https://ezid.cdlib.org/manage/display_xml/doi:10.5072/FK2V69J19C) it has the creator's affiliation as Oregon State University. When I created the plan I was affiliated with UCNRS.  
- [x] I tested the collaborators connection w/ the DOI record and I'm still having an issue with one of the users (me) being listed as Oregon State University. For example, [this plan](https://dmptool-stg.cdlib.org/plans/58910) and this [DataCite record](https://ezid.cdlib.org/manage/display_xml/doi:10.5072/FK28P63Z88). If you're not seeing this issue on your end maybe this is just something off with my profile on staging and not a real issue. I'm not sure so adding it here. 
- [x] It looks like the fundref ID is not being passed to DataCite. For example [this record](https://ezid.cdlib.org/manage/display_xml/doi:10.5072/FK28P63Z88) [x] Maria
- [x] Change 'a few minutes' to 'a few seconds' [x] Maria
- [x] Send the plan visibility to the DMPHub so it can hide the link back to the DMP
- [x] On the Finalize/Publish tab the "Save" and "Register" buttons should be evenly aligned. Right now the "Save" button is a few spaces to the right. [x] Maria
- [ ] The collaborators listed in a DMP do not match what is in the DataCite xml record. For example, this plan (https://dmptool-stg.cdlib.org/plans/58891/contributors) has one contributor (Maria Praetzellis | ORCID: 440012999999999999) but the xml includes Diana Ester LaCalle ORCID: 440012999999999999). https://ezid.cdlib.org/manage/display_xml/doi:10.5072/FK2GX4DH6W
I made a new plan to test this again and found the same thing where the contributors do not match btwn the DMPTool & xml. It also has the creator of the DMP as "Diana Ester LaCalle". I'm not sure why this name is sticking, I have used it in prior plans but not this one.
Plan: https://dmptool-stg.cdlib.org/plans/58916
XML: https://ezid.cdlib.org/manage/display_xml/doi:10.5072/FK2C53PQ2H

**ID Landing Page/DMPHub**
- [ ] When a plan is publicly available the link at the top right of the page should point to the PDF. It currently goes to the API: https://dmptool-stg.cdlib.org/api/v1/plans/58910

**Comment by briri** [comment]
Add ethics to JSON

**Comment by briri** [comment]
@mariapraetzellis I cannot duplicate the issue with the Funding Reference Number. There used to be code in place that put the Plan id in there but that has been removed. If its an old plan, the value may have already been there due to that old code.

The issue with the missing Fundref info in the DataCite record was due to bad data in the stage DMPHub database. I just ran a script that went out and harvested the Fundref ids from ROR. I created a script that will force a sync of EZID's copy record.
